ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -219,6 +219,10 @@ if tflite2_support_is_available
   tflite2_compile_args += '-DTFLITE_FLOAT16=1'
   tflite2_compile_args += '-DTFLITE_COMPLEX64=1'
 
+  if cc.has_header('tensorflow2/lite/delegates/xnnpack/xnnpack_delegate.h') or cc.has_header('tensorflow/lite/delegates/xnnpack/xnnpack_delegate.h')
+    tflite2_compile_args += '-DTFLITE_XNNPACK_DELEGATE_SUPPORTED'
+  endif
+
```

Comment:
Let's define ```tflite2-xnnpack-delegate-support``` with ```type: 'feature'``` and run 222-224 according to the given options (auto, disabled, enabled)

Reply:
This is to provide controllability for platform build and integration.

Reply:
As Tizen would use tf2lite which supports XNNPACK, I set the option as `true` and let the meson check the header for it during the build configuration. PTAL

Reply:
with type:'feature'

if value=enabled
   error if header does not exist
   enable if exist
elseif value=auto
   enable if exist
   disable if not exist
else
   disable